src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs

```diff
@@ -149,5 +151,866 @@ public void TFS_470021()
                 Assert.Equal(0, errorCount);
             }
         }
+
+        /// <summary>
+        /// Test for issue #30218, resource Sch_MinLengthGtBaseMinLength
+        /// </summary>
+        [Fact]
+        public void MinLengthGtBaseMinLength_Throws()
```

Comment:
```suggestion
        public void MinLengthLtBaseMinLength_Throws()
```

Reply:
Fixed.  I got thrown off by the name of the corresponding string resource containing "Gt".  I also noticed that I had put "greater than" in the reworded string, and fixed that too.